generator/source/faq/videos.rst

```diff
@@ -22,7 +22,10 @@ Selected YouTube Videos
 
 2019 YouTube Videos
 -------------------
-Hyperledger Sawtooth Performance Metrics with Grafana (Dan Anderson, Intel, 2019)
+Hyperledger Sawtooth Application Development Overview using "SimpleWallet" (Dan Anderson, Intel, 2019)
```

Comment:
Small nitpick: There are two authors shown at the start of the video: you and Tom.  To be totally consistent, you could add his name here. But this isn't a blocker.

Reply:
Thanks for the review.

I fixed the name and rebased.

The names with videos are the speakers.